ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -186,7 +188,7 @@ export default class WMWorkerPoolEditor extends Component {
     return (
       !this.state.invalidProviderConfig &&
       !Object.values(validation).some(({ error }) => Boolean(error)) &&
-      providerId !== ''
+      providerId !== '' && this.state.validation.owner.check
```

Comment:
We can probably fix this without having to introduce a new state `check`. Here's one way:

```diff
diff --git a/ui/src/components/WMWorkerPoolEditor/index.jsx b/ui/src/components/WMWorkerPoolEditor/index.jsx
index 26f865850..f7d9485d5 100644
--- a/ui/src/components/WMWorkerPoolEditor/index.jsx
+++ b/ui/src/components/WMWorkerPoolEditor/index.jsx
@@ -182,14 +182,16 @@ export default class WMWorkerPoolEditor extends Component {
 
   isValid() {
     const {
-      workerPool: { providerId },
+      workerPool: { workerPoolId1, workerPoolId2, owner },
       validation,
     } = this.state;
 
     return (
       !this.state.invalidProviderConfig &&
       !Object.values(validation).some(({ error }) => Boolean(error)) &&
-      providerId !== ''
+      workerPoolId1 !== '' &&
+      workerPoolId2 !== '' &&
+      owner !== ''
     );
   }
```

Reply:
I have resolved the issue . Thanks for the feedback!